scio-test/src/main/scala/com/spotify/scio/testing/TransformOverride.scala

```diff
@@ -105,6 +98,42 @@ object TransformOverride {
     PTransformOverride.of(new EqualNamePTransformMatcher(name), overrideFactory)
   }
 
+  /**
+   * @return
+   *   A [[PTransformOverride]] which when applied will override a [[PTransform]] with name `name`
+   *   with a transform flat-mapping elements via `fn`.
+   */
+  def off[T: ClassTag, U](name: String, fn: T => Iterable[U]): PTransformOverride = {
```

Comment:
To be fixed: when I use the same function name `of`, I get `double definition` because it will have the same signature with the previous function after erasure:
```
have same type after erasure: (name: String, fn: Function1, evidence$1: scala.reflect.ClassTag): org.apache.beam.sdk.runners.PTransformOverride
```

Reply:
Solved: chose new name `ofIter`

Reply:
Using a separate name is a good choice, `ofN` or `ofSeq` might be other options but `ofIter` is reasonable.